src/internal/nativeInterface.ts

```diff
@@ -21,7 +21,7 @@ if (!RNDeviceInfo) {
     throw new Error(`@react-native-community/react-native-device-info: NativeModule.RNDeviceInfo is null. To fix this issue try these steps:
   • For react-native <= 0.59: Run \`react-native link react-native-device-info\` in the project root.
   • Rebuild and re-run the app.
-  • If you are using CocoaPods on iOS, run \`pod install\` in the \`ios\` directory and then rebuild and re-run the app. You may also need to re-open Xcode to get the new pods.
+  • If you are using CocoaPods on iOS or macOS, run \`pod install\` in the \`ios\` or \`macos\` directory and then rebuild and re-run the app. You may also need to re-open Xcode to get the new pods.
```

Comment:
```suggestion
  • If you are using CocoaPods for apple targets, run \`pod install\` in the \`macos\` directory, then rebuild and re-run the app. You may also need to re-open Xcode to get the new pods.
```

I think this is more accurate? I'm always iffy on wording and documentation though, feel free to mash the words around, but I don't think the ios directory will exist so that at least should be removed